src/nxdoc/web-ui/web-ui-tutorials/how-to-create-and-customize-nuxeo-actions-menu.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'HOWTO: Create a Nuxeo actions menu'
+review:
+    comment: ''
+    date: '2020-03-25'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to create a Nuxeo actions menu.
+        level: Advanced
+        tool: code
+        topics: Web UI
+labels:
+    - lts2017-ok
+    - lts2019-ok
+    - tutorial
+    - nuxeo-elements
+    - nuxeo-ui-elements
+    - polymer
+tree_item_index: 500
+---
+
+{{! excerpt}}
+In this tutorial you will learn how to create and customize a Nuxeo actions menu on your code. The actions menu is a responsive menu that can hold actions (e.g. buttons), which hides behind a dropdown according to the available (or predefined) space.
+{{! /excerpt}}
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/HOWTO: Create a Nuxeo actions menu/nuxeo actions menu screenshot
+    name: nuxeo-action-menu.png
+    1.1.3#screenshot#up_to_date
+--}}
+![nuxeo actions menu screenshot](nx_asset://9c147cdc-311a-4190-b7d1-7d2a255216f2 ?w=300,border=true)
+
+## Create an actions menu:
+First, we start by adding a nuxeo actions menu element.
+
+```
+<nuxeo-actions-menu>
+</nuxeo-actions-menu>
+```
+
+Then we add the menu items, which should consist of actions - such as a nuxeo defined action, for example:
+
+```
+<nuxeo-add-to-collection-button document="[[document]]">
```

Comment:
It would help to also see the nuxeo-actions-menu tags, so we have an idea of how the whole thing would look like.

Reply:
thanks! suggestion added